website/assets/styles/pages/articles/basic-webinar.less

```
@@ -0,0 +1,749 @@
+      font-style: normal;
+      font-weight: 800;
+      line-height: 120%;
+      margin-top: 40px;
+      margin-bottom: 32px;
+    }
+    h4 {
+      font-size: 16px;
+      margin-top: 24px;
+      margin-bottom: 24px;
+    }
+    h2 > code:not(.bash):not(.nohighlight):not(.mermaid) {
```

Stylelint (17.7.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)